modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/manager/DSEnvelopeManagerImpl.java

```diff
@@ -127,6 +127,37 @@ public DSEnvelope getDSEnvelope(long groupId, String dsEnvelopeId) {
 			evenlopeJSONObject -> _toDSEnvelope(evenlopeJSONObject), _log);
 	}
 
+	@Override
+	public JSONObject getDSEnvelopesJSONObject(
+		long groupId, String fromDateString, long count, long startPosition) {
```

Comment:
All the parameters received from Requests are String, in `GetDSEnvelopesMVCResourceCommand.java`  you are parsing some fields like `count` and `startPosition` to long, and then at line 137 you are parsing again to String

We could use all as string, if we don't need the value as Long

What you think?

Reply:
As mentioned, I agree with the changes and I'm already making them.

Reply:
I agree, but now this will be necessary to calculate the requested paging implementation.